README.md

```diff
@@ -39,30 +39,118 @@ extern crate rss;
 
 ## Reading
 
-A channel can be read from any object that implements the `BufRead` trait. 
+### From a `BufRead`
+
+A channel can be read from any object that implements the `BufRead` trait.
 
 ```rust
-let reader: BufRead = ...;
+use std::fs::File;
+use std::io::BufReader;
+use rss::Channel;
+
+let file = File::open("tests/data/rss2sample.xml").unwrap();
+let reader = BufReader::new(file);
 let channel = Channel::read_from(reader).unwrap();
 ```
 
+### From a URL
+
+A channel can also be read from a URL.
+
+To enable this functionality you must enable the `from_url` feature in your Cargo.toml.
+
+```toml
+[dependencies]
+rss = { version = "*", features = ["from_url"] }
+```
+
+```ignore
```

Comment:
though, since these code blocks aren't being tested (since they're in the README), it actually doesn't matter if we `no_run` or `ignore`, so yeah, just `rust` should be fine